data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -406,7 +406,7 @@
     "//": "Standard set of clothes worn by a military helicopter pilot.",
     "items": [
       { "item": "chestrig", "prob": 40 },
-      { "item": "flight_helmet", "ammo-item": "light_plus_battery_cell", "charges": [ 0, 150 ] },
+      { "item": "flight_helmet", "ammo-item": "light_battery_cell", "charges": [ 0, 150 ] },
```

Comment:
shouldn`t those charges be `"charges": [ 0, 10 ]` , since light_battery_cell holds 10, not 150?

Reply:
It will cut the battery charge automatically to 0 - 16